yocto/arm64/colibri-imx8x/metas

```diff
@@ -0,0 +1,16 @@
+meta-openembedded/meta-oe
+poky/meta-poky
+meta-openembedded/meta-python
+meta-openembedded/meta-networking
+meta-openembedded/meta-perl
+meta-openembedded/meta-filesystems
+meta-freescale
+meta-freescale-3rdparty
+meta-freescale-distro
+meta-virtualization
+meta-selinux
```

Comment:
we just removed that layer, you should also drop meta-selinux here

Reply:
Fixed